src/kafka-util/src/client.rs

```diff
@@ -301,7 +301,7 @@ where
                         Some(port) => port.to_string(),
                     },
                 };
-                info!(
+                debug!(
```

Comment:
Similar to comment about other downgrade: this was a hard-won log line that's come in really handy in debugging production issues. I've never seen it be terribly spammy, as it only fires when librdkafka resolves DNS, which is not that frequently—and it's often a good signal that something is up, if librdkafka is re-resolving DNS.

Reply:
if the ssh tunnel cant be created, then rdkafka has a very hot loop where it does this, but thats on a rdkafka thread, so many we leave it for now?